### ISSUE TITLE
Sources marked changed

### ISSUE THREAD

**Issue by tschmidtb51** [ISDuBA/ISDuBA#649]
All sources were marked "changed" after I built ISDuBA today (currently on `0.5.0`, before on `0.4.0-100-gaa4310cd`). Was that intended?

**Comment by s-l-teichmann** [comment]
Yes, kind of .. there where a bug in calculating the checksums (which we use to track changes) over aggregators and  sources.  The result were wrong checksums in the database.

https://github.com/ISDuBA/ISDuBA/commit/873dec67a43c80cc45d6abd1b058363f8e8f6f4a

**Comment by koplas** [comment]
Closing because the checksum calculation was fixed with v0.5.0.
